pkg/reconciler/trustedresources/sign.go

```diff
@@ -19,14 +19,16 @@ package trustedresources
 import (
 	"bytes"
 	"crypto/sha256"
+	"encoding/base64"
 	"encoding/json"
 	"fmt"
 
 	"github.com/sigstore/sigstore/pkg/signature"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
 )
 
-// signInterface returns the encoded signature for the given object.
-func signInterface(signer signature.Signer, i interface{}) ([]byte, error) {
+// SignInterface returns the encoded signature for the given object.
+func SignInterface(signer signature.Signer, i interface{}) ([]byte, error) {
```

Comment:
does this need to be exported? All usages seem within the package?

Reply:
Yes it shouldn't be exported. There were some functions using it but they also removed now. Change this back